<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Create_Image_Info</class>
 <widget class="QDialog" name="Create_Image_Info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Creating Images</string>
  </property>
  <property name="windowIcon">
   <iconset resource="EXE_Icons.qrc">
    <normaloff>:/exe_icon/EXE Icons/FRD_icon.png</normaloff>:/exe_icon/EXE Icons/FRD_icon.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>461</width>
     <height>23</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedKingdom"/>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_Creating">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Creating Images...</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>431</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_now">
      <property name="text">
       <string>Now:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_progress">
      <property name="text">
       <string>Progress:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_speed">
      <property name="text">
       <string>Speed:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_leftTime">
      <property name="text">
       <string>Estimated time left:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_timeUsed">
      <property name="text">
       <string>Time used:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="EXE_Icons.qrc"/>
 </resources>
 <connections/>
</ui>
